src/CarSearch/CarList.js

```diff
@@ -1,39 +1,64 @@
+/* global google */
+
 import React, { PropTypes } from 'react';
 import { List } from 'react-virtualized';
-import { connect } from '../util/react-redux-custom-store-key';
+import classnames from 'classnames';
+import { connect } from './util/react-redux-custom-store-key';
+import { changeSelectedLocation } from './actions';
+import { getDistanceMeters, getDistanceString } from './util/distance';
 import './styles/CarList.css';
 
-const mapStateToProps = ({ serverData, mapBounds }) => ({
+const mapStateToProps = ({ serverData, mapBounds, selectedLocation, selectedPlace }) => ({
   data: serverData.data,
-  mapBounds
+  mapBounds,
+  selectedLocation,
+  selectedPlace
 });
 
-const CarList = ({ data, mapBounds }) => {
+const CarList = ({ data, mapBounds, selectedLocation, selectedPlace }, { carSearchStore }) => {
   if (!data) {
-    return <div>Loading</div>;
+    return <div>Loading...</div>;
   }
 
   const locations = [];
+  let selectedLocationIndex;
   for (const city of data.citiesAndLocations) {
     for (const location of city.locations) {
-      if (mapBounds.contains({ lat: location.geo[0], lng: location.geo[1] })) {
+      const latLng = new google.maps.LatLng(location.geo[0], location.geo[1]);
+      if (mapBounds && mapBounds.contains(latLng)) {
+        if (location === selectedLocation) {
+          selectedLocationIndex = locations.length;
+        }
+        location.city = city;
+        location.distanceMeters = selectedPlace
+          ? getDistanceMeters(selectedPlace.geometry.location, latLng) : null;
         locations.push(location);
       }
     }
   }
+  if (selectedPlace) {
+    locations.sort((a, b) => (a.distanceMeters - b.distanceMeters));
+  }
 
   // eslint-disable-next-line
-  const rowRenderer = ({ key, index, style }) => (
-    <div
-      className="CarListItemContainer"
-      key={key}
-      style={style}
-    >
-      <div className="CarListItem">
-        {locations[index].addr}
+  const rowRenderer = ({ key, index, style }) => {
+    const location = locations[index];
+
+    return (
+      // eslint-disable-next-line
```

Comment:
Why are these inlined? Is there a reason the next line throws an error which we can't catch in global rules?